### PR TITLE
README Updated

### PR DESCRIPTION
**Pull request by JonasHedEng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# 2018-project-myfirstline
+# MyFirstLine
+MyFirstLine is a project aimed at supporting teachers tutor students in programming as well as make it easier for students to learn from each other.
+
+Each user may create a task which contains a description of how the code should work, test input and expected output for the corresponding input.
+
+To create a task or solve a task, the user needs to be logged in. To view a task, no login is needed.
+
+When a user has solved a task, he/she may "Like" other solutions if the author of the solution chose to "Publicize" their solution.
+The user who created the task may also see a list of all the users whom have solved the task(a special user type "teacher", may see all solutions and not just which users have solved it).
```
Comment (willeponken) [inline README.md]: Missing space:
> task(a

:dancing_men:

Comment (JonasHedEng) [reply README.md]: Yeyeyeye

File: README.md [hunk 2]
```diff
@@ -1 +1,13 @@
...
+Each user may create a task which contains a description of how the code should work, test input and expected output for the corresponding input.
+
+To create a task or solve a task, the user needs to be logged in. To view a task, no login is needed.
+
+When a user has solved a task, he/she may "Like" other solutions if the author of the solution chose to "Publicize" their solution.
+The user who created the task may also see a list of all the users whom have solved the task (a special user type "teacher", may see all solutions and not just which users have solved it).
+
+Thus, this can be used by teachers to make assignments as well as students can create challenges with solutions to each other for learning.
+
+Thus, this can be used by teachers to make assignments as well as students can create challenges with solutions to each other for learning.
```
Comment (willeponken) [inline README.md]: Isn't this duplicate? :thinking: 

I really like the idea, good luck! :smiley: :snowflake: :dollar:

Comment (JonasHedEng) [reply README.md]: It is! Thanks for noticing! :smile:

Comment (willeponken) [reply README.md]: Happy to help :dancer:

**Comment by JonasHedEng** [comment]
Solves #1

**Comment by JonasHedEng** [comment]
Resolves #1

**Pull request merged**